setup.py

```diff
@@ -22,28 +48,13 @@
         ],
     },
     install_requires=[
-        "gym[atari,box2d]",     # common environments
-        "numpy",                # math library
-        "matplotlib",           # plotting library
-        "opencv-python>=3.,<4.",# used by atari wrappers
-        "pybullet",             # continuous environments
-        "tensorboardX",         # tensorboard compatibility
+        "gym>=0.17.2",             # common environment interface
+        "numpy>=1.18.0",           # math library
+        "matplotlib>=3.3.0",       # plotting library
+        "opencv-python>=3.,<4.",   # used by atari wrappers
+        "torch>=1.5.1,<1.6",       # core deep learning library
+        "tensorboard>=2.3.0",      # logging and visualization
```

Comment:
why both tensorboard and tensorboardx?

Reply:
tensorboardX only provides an interface for writing the event files. You still need tensorboard to view them.